samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md

```diff
@@ -1,25 +1,25 @@
 # Receive events with the Event Processor Host in .NET Standard with a custom role which grants Listen claim for Event Hubs and blob claims for Storage accounts.
 
+**Note:** The sample in this folder uses the legacy Event Hubs library `Microsoft.Azure.EventHubs`. We strongly recommend you to use the current library `Azure.Messaging.EventHubs`. This file has instructions for both.
 
-## Run the sample
-
-**Note:** This sample uses the legacy Event Hubs library `Microsoft.Azure.EventHubs`. We strongly recommend you to use the current library `Azure.Messaging.EventHubs`. 
+## Prerequisites
 
-See [sample](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/samples/DefiningCustomCredentialTypes.md#authenticating-with-the-on-behalf-of-flow) that uses the new library to authenticate after the environment variables for client id and secret are set. 
```

Comment:
Was there any particular reason to talk about "behalf of flow" here?

Reply:
I think I was trying to find a corresponding T2 sample to point to.

Reply:
ok, I was wondering if @jsquire had any particular reason to talk about the "behalf of flow" due to the custom role set up being explained in this sample

Reply:
I would question why we're pointing at the custom credential sample.  Unless I'm missing a nuance, the standard [ClientSecretCredential](https://docs.microsoft.com/dotnet/api/azure.identity.clientsecretcredential) would probably be the more relevant type; the custom role would likely be assigned to a normal service principal.

Reply:
I talked to Lily offline and it looked like she took a best guess at what would fit best. 
@jsquire Can you take a look at the text under "Run the sample for current library Azure.Messaging.EventHubs" section in this file and see if that is good enough?

Reply:
I don't see that specific text, but the link that is used to point to our AAD sample is correct.   I don't like our text there for T2, but I'll do a quick update there.